pkg/tnf/identifier/identifier.go

```diff
@@ -85,3 +88,31 @@ func (i *Identifier) UnmarshalJSON(b []byte) error {
 
 	return i.unmarshalSemanticVersion(objMap)
 }
+
+// XformToGinkgoItIdentifier transform the Identifier into a test Id that can be used to skip
+// specific tests
+func XformToGinkgoItIdentifier(identifier Identifier) string {
```

Comment:
As we discussed, test building blocks have nothing to do with ginkgo or It. Just call it something like GetShortName()

Reply:
Renamed, it had no sense the previous name. Neither had sense to have a function pointing to names to skip tests.